lib/web_ui/lib/src/engine/text_editing/text_editing.dart

```diff
@@ -1351,10 +1352,10 @@ class EditableTextGeometry {
   }
 
   /// The width of the editable in local coordinates, i.e. before applying [globalTransform].
-  final double width;
```

Comment:
both width and height should not be null.

The values are coming from this code in flutter framework: https://github.com/flutter/flutter/blob/4d7525f05c05a6df0b29396bc9eb78c3bf1e9f89/packages/flutter/lib/src/services/text_input.dart#L885

Reply:
Done.